include/openPMD/IO/HDF5/HDF5IOHandlerImpl.hpp

```diff
@@ -70,6 +70,9 @@ namespace openPMD
 
         hid_t m_datasetTransferProperty;
         hid_t m_fileAccessProperty;
+        hid_t m_fileCreateProperty;
+
+        hbool_t m_hdf5_collective_metadata;
```

Comment:
```suggestion
        hbool_t m_hdf5_collective_metadata = 0;
```
or, if default-on is what we want (see below)
```suggestion
        hbool_t m_hdf5_collective_metadata = 1;
```